### DISCUSSION TITLE
Access to push changes

### DISCUSSION THREAD

**Discussion by mikko-pulli** [FantasticFiasco/serilog-sinks-http#399] [General]
Hi

I am interested in contributing to the repository, specifically to work on a new sink to send over http and only on any failure store to file. This would save on write cycles on edge computers

Would it be possible to grant me write access to the repository? My GitHub username is mikko-pulli.

Thank you for considering my request.

Best regards,
Mikko Pulli

**Comment by FantasticFiasco** [comment]
Hi @mikko-pulli.

I'm currently in a state where I cant accept major contributions. What I propose you to do is to fork the repository and apply your changes to the fork.
